src/WinUIExSample/MainWindow.xaml

```diff
@@ -20,47 +22,145 @@
+                <RadioButton Content="BottomRight" Checked="moveWindowRadioButtons_Checked"/>
+                </RadioButtons>-->
+                <TextBlock Text="Move Window To:" />
+                <Grid ColumnDefinitions="*,*,*" RowDefinitions="*,*,*">
+                    <Button Click="moveWindowButton_Clicked" Content="TopLeft" />
+                    <Button
+                        Grid.Column="1"
+                        Click="moveWindowButton_Clicked"
+                        Content="TopCenter" />
+                    <Button
+                        Grid.Column="2"
+                        Click="moveWindowButton_Clicked"
+                        Content="TopRight" />
+
+                    <Button
+                        Grid.Row="1"
+                        Click="moveWindowButton_Clicked"
+                        Content="MiddleLeft" />
+                    <Button
+                        Grid.Row="1"
+                        Grid.Column="1"
+                        Click="moveWindowButton_Clicked"
+                        Content="MiddleCenter" />
+                    <Button
+                        Grid.Row="1"
+                        Grid.Column="2"
+                        Click="moveWindowButton_Clicked"
+                        Content="MiddleRight" />
+
+                    <Button
+                        Grid.Row="2"
+                        Click="moveWindowButton_Clicked"
+                        Content="BottomLeft" />
+                    <Button
+                        Grid.Row="2"
+                        Grid.Column="1"
+                        Click="moveWindowButton_Clicked"
+                        Content="BottomCenter" />
+                    <Button
+                        Grid.Row="2"
+                        Grid.Column="2"
+                        Click="moveWindowButton_Clicked"
+                        Content="BottomRight" />
+
+                </Grid>
+
+
+                <StackPanel Orientation="Horizontal" HorizontalAlignment="Center">
+                    <TextBlock VerticalAlignment="Center" Text="X: "/>
+                    <NumberBox
+                        x:Name="x_box"
+                        Width="100"
+                        Value="300" 
+                        Margin="5"/>
+                    <TextBlock VerticalAlignment="Center" Text="Y: "/>
+                    <NumberBox
+                        x:Name="y_box"
+                        Width="100"
+                        Value="300" 
+                        Margin="5"/>
+                </StackPanel>
+                <StackPanel Orientation="Horizontal" HorizontalAlignment="Center">
+                <Button Click="MoveToPoint_Click" Content="Move to XY" />
+                <Button Click="CentreOnPoint_Click" Content="Centre on XY" />                    
+                </StackPanel>
+
+            </StackPanel>
+        </ScrollViewer>
```

Comment:
In general don't make PRs that makes unrelated changes - it makes it near impossible to review what you actually changes, and all these reordering and reformatting changes aren't necessary and might not match the maintainer's preference. Could you please undo all those so only the things you added show up?

Reply:
Will do.